<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RGB_SearchClass</class>
 <widget class="QDialog" name="RGB_SearchClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RGB_Search</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qtcolortable.qrc">
    <normaloff>:/QtColorTable/Palette.ico</normaloff>:/QtColorTable/Palette.ico</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter Color Name for search:</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>561</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="searchBtn">
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closeBtn">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>341</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Red</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="redSpin"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Green</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="greenSpin"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Blue</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="blueSpin"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtcolortable.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>RGB_SearchClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
